### PR TITLE
Fix the ESH UI compatibility issues

### PR DESCRIPTION
**Pull request by nagdeepgk**
uwc/build_scripts : Fix the ESH UI compatibility issues

    1. Replace "read -p <string> var_name" with 2 separate "echo & read" statements.

    Signed-off-by: nagdeepgk <nagdeep.gk@intel.com>

### CODE REVIEW

**Comment by nagdeepgk** [comment]
@vkb1  @sindujaks @aditisa1 @vats-in Can you Kindly review this.

File: build_scripts/02_provision_UWC.sh [hunk 1]
```diff
@@ -412,8 +413,13 @@ parse_command_line_args()
...
                             2: All basic UWC modules + KPI-tactic Application (no SPARKPLUG-BRIDGE)
                             3: All modules (with KPI-tactic Application and with SPARKPLUG-BRIDGE)
-                            4: All basic UWC modules + SPARKPLUG-BRIDGE (no KPI-tactic Application)"
-        read -p "Enter the recipe (e.g. --recipe=1 or 2 or 3 or 4 ):"  recipe 
+                            4: All basic UWC modules + SPARKPLUG-BRIDGE (no KPI-tactic Application)
+                            5: Basic UWC micro-services with Time series micro-services (Telegraf & InfluxDBConnector)
+                            6: Running Basic UWC micro-services with time series services (Telegraf & InfluxDBCOnnector) along with KPI-tactic app
+                            7: Running Basic UWC micro-services with time series services (Telegraf & InfluxDBCOnnector) along with Sparkplug service
+                            8: Running the sample DB publisher with Telegraf, InfluxDBCOnnector, ZmqBroker & Etcd container"
+        echo "Enter the recipe (e.g. --recipe=1 or 2 or 3 or 4 or 5 or 6 or 7 or 8 ):"
```
Comment (vkb1) [inline build_scripts/02_provision_UWC.sh:421]: Indentation is bit off here

File: build_scripts/2.1_ConfigureScada.sh [hunk 2]
```diff
@@ -187,26 +187,30 @@ get_user_inputs()
...
 		if [ -z $TLS ];then
 			echo "${RED}Empty value entered.${NC}"
 		elif [ $TLS != "yes" ] && [ $TLS != "no" ] ;then
 			echo "${RED}Invalid value entered. Allowed values (yes/no)${NC}"
 		elif [ $TLS == "yes" ];then
 			IS_TLS=true
-			read -p "Enter the CA certificate full path including file name (e.g. /home/certs/root-ca.crt):" CA 
+			echo "Enter the CA certificate full path including file name (e.g. /home/certs/root-ca.crt):"
+			read CA 
 				if [ -z $CA ]; then
```
Comment (vkb1) [inline build_scripts/2.1_ConfigureScada.sh:200]: Indentation is bit off here.

**Review by vkb1 — APPROVED** [review]
The commit title and description can be something on below lines:

```
uwc/build_scripts: Fixed the ESH UWC installer compatibility issue

Fixed the issue of ESH UWC installer not working for few use cases
due to usage of `read -p`, fixed it by prompting the message and
reading that into a variable

Singed-off-by:...
```

**Review by aditisa1 — APPROVED** [review]

**Review by sindujaks — APPROVED** [review]

File: build_scripts/02_provision_UWC.sh (hunk 1 above)
Comment (nagdeepgk) [reply build_scripts/02_provision_UWC.sh:421]: actually it looks like this which is ifne right. echo in lin eiwth other echo/reads. @vkb1 
image

File: build_scripts/2.1_ConfigureScada.sh (hunk 2 above)
Comment (nagdeepgk) [reply build_scripts/2.1_ConfigureScada.sh:200]: the indentation with surrounding lines.

image

**Pull request merged**